const throwError = () => {
  throw new Error("This is my error foo!!!");
};

export default throwError;
